dev_guide/configmaps.adoc

```diff
@@ -169,6 +178,12 @@ You can also pass the `--from-file` option with a specific file, and pass it
 multiple times to the CLI. The following yields equivalent results to the
 xref:configmaps-creating-from-directories[Creating from Directories] example:
 
+[NOTE]
+====
+If you create a configmap from a file, you can include files containing non-UTF8 data will be placed in this new field in order to preserve the non-UTF8 data. 
```

Comment:
Please double-check this sentence. Maybe s/containing non-UTF8 data will be placed/that contain non-UTF8

Reply:
@kalexand-rh That sentence came from the [Kubernetes change log](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.10.md#apps).

I changed it to _on-UTF8 data will be placed in this new field without corrupting the non-UTF8 data._, taken from https://github.com/kubernetes/kubernetes/pull/57899#issue-161357094